src/style.css

```diff
@@ -1 +1,84 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc');
+  font-weight: bold;
+}
+
+:root {
+  --font-family: Avenir, Arial, sans-serif;
+  --font-color: #3d4e61;
+  --background-color: #fff;
+  --border-color: #e1e7ed;
+  --box-shadow: rgba(61, 78, 97, 0.1);
+  --hover-box-shadow: rgba(61, 78, 97, 0.2);
+  --placeholder-color: #3d4e61;
+  --search-icon-size: 19px;
+  --search-icon-small-size: 11px;
+  --search-bar-height: 70px;
+  --search-bar-small-height: 42px;
+  --font-size: 16px;
+  --font-size-small: 14px;
+}
+
+.page__body {
+  padding: 0;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  font-family: var(--font-family);
+  font-size: var(--font-size);
+  color: var(--font-color);
+  background-color: var(--background-color);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: var(--search-icon-size);
+  background-position: 25px;
+  width: 100%;
+  height: var(--search-bar-height);
+  padding-left: 62px;
+  box-shadow: 0 1px 8px var(--box-shadow);
+  outline: none;
+}
+
+.search-bar__input::placeholder {
+  color: var(--placeholder-color);
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px var(--hover-box-shadow);
+}
+
+.search-bar__input:focus {
+  font-weight: bold;
+  background-position: 25px;
+  box-shadow: 0 3px 8px var(--hover-box-shadow);
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__input--small {
```

Comment:
The `.search-bar__input--small` class correctly overrides the necessary properties for the smaller search bar variant, maintaining consistency in the design. However, make sure that the `font-weight: 400;` is necessary here, as the normal font weight is usually the default and might not need to be specified unless it's overriding a previous setting.